src/doom/deh_thing.c

```diff
@@ -95,6 +95,7 @@ DEH_BEGIN_MAPPING(thing_mapping, mobjinfo_t)
   DEH_MAPPING("Action sound",        activesound)
   DEH_MAPPING("Bits",                flags)
   DEH_MAPPING("Respawn frame",       raisestate)
+  DEH_MAPPING("Dropped item",        droppeditem)
```

Comment:
This needs a `[crispy]` comment, please.

Reply:
I added the comment but above (so github didn't detect it as change).
Is that ok or should it be in the same line?

Reply:
That's alright here.